skins/universal/dark/dark.skn

```diff
@@ -1,4 +1,4 @@
-<skin version="1.3" author="VitoZz" name="Dark" path="C:/Users/vital/AppData/Roaming/Psi+/skins">
```

Comment:
for the future, no need to delete this path, it will reappear with each new skin release, it is always added by the plugin when saving the skin